incubator/redis-cache/values.yaml

```diff
@@ -0,0 +1,24 @@
+# Default values for redis-cache.
```

Comment:
Please make resources and image pull policy configurable for all containers and don't specify default resources. Again, please look at this example https://github.com/kubernetes/charts/blob/master/stable/nginx-ingress/values.yaml.